steerable/SCFpyr_PyTorch.py

```diff
@@ -156,23 +160,17 @@ def _build_levels(self, lodft, log_rad, angle, Xrcos, Yrcos, height):
             for b in range(self.nbands):
 
                 anglemask = pointOp(angle, Ycosn, self.Xcosn + np.pi*b/self.nbands)
-                anglemask = anglemask[None,:,:,None]  # for broadcasting
+                anglemask = anglemask[None,:,:]  # for broadcasting
                 anglemask = torch.from_numpy(anglemask).float().to(self.device)
 
                 # Bandpass filtering                
                 banddft = lodft * anglemask * himask
```

Comment:
should it be:

banddft=(-1j)**order*lodft *anglemask * himask ? 

As the [code here](https://github.com/LabForComputationalVision/pyrtools/blob/47c8c87e563c4f0b95fae28387977f390609a797/pyrtools/pyramids/SteerablePyramidFreq.py#L169)